SETUP.md

```diff
@@ -29,8 +29,6 @@ You can do Code.org development using macOS, Ubuntu, or Windows (running Ubuntu
 
 1. `git lfs pull`
 
-1. `gem install bundler -v 2.3.22 && rbenv rehash`
```

Comment:
See: https://github.com/code-dot-org/code-dot-org/pull/60329/files#r1712635193 for discussion on removing low-level gem installation of a bundler version: its already installed high-level by bundle install.